integration/test/test_hint_time.cpp

```diff
@@ -22,6 +23,7 @@ int main(int argc, char **argv)
     int comm_rank;
     MPI_Init(&argc, &argv);
     MPI_Comm_rank(MPI_COMM_WORLD, &comm_rank);
+    sleep(5);
```

Comment:
Is this still required?  If yes, we need to add some documentation somewhere that makes it clear that if you try to profile regions that occur too quickly after app startup, data will be lost.

Reply:
The data isn't "lost" we do not collect hardware samples until the controller is ready, but the app runs immediately.  For this reason, we cannot associate hardware metrics with the application data that *is* collected and processed (time stamped application data is not lost).  So, for tests that require hardware data to be collected in regions that occur early in the app, the sleep is required.